turf/src/commonTest/kotlin/io/github/dellisd/spatialk/turf/BooleansTests.kt

```diff
@@ -0,0 +1,221 @@
+package io.github.dellisd.spatialk.turf
+
+import io.github.dellisd.spatialk.geojson.Feature.Companion.toFeature
+import io.github.dellisd.spatialk.geojson.MultiPolygon
+import io.github.dellisd.spatialk.geojson.Point
+import io.github.dellisd.spatialk.geojson.Polygon
+import io.github.dellisd.spatialk.turf.utils.readResource
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+@ExperimentalSerializationApi
+class BooleansTests {
+
+    @Test
+    fun testFeatureCollection() {
+        // test for a simple polygon
+        val poly = Polygon(arrayOf(
+            arrayOf(
+                doubleArrayOf(0.0, 0.0),
+                doubleArrayOf(0.0, 100.0),
+                doubleArrayOf(100.0, 0.0),
+                doubleArrayOf(0.0, 0.0)
+            )
+        ))
+        val ptIn = Point(doubleArrayOf(50.0, 50.0))
```

Comment:
Once the boolean functions have been updated to accept `Position`, these can be replaced with the `lngLat` helper function.
```suggestion
        val ptIn = lngLat(50.0, 50.0)
```